### ISSUE TITLE
How can u convert the model to your rename layer?

### ISSUE THREAD

**Issue by toddwyl** [oraoto/paddle-vgg16#1]
I want to convert vgg19 by using caffe2fluid.
But I cannot convert to the param by my rename layer.
Can you share your convert.py for me?
[vgg19-pd_my_link](https://github.com/strawsuper/vgg19_paddlepaddle)

**Comment by toddwyl** [comment]
I want to use like your way:
```
from vgg16 import VGG16
import paddle.fluid as fluid

img = fluid.layers.data('img', shape=[3, 224, 224])
predict = VGG16(include_top=True, infer=True).net(img)

exe = fluid.Executor(fluid.CPUPlace())
# All params are prefiexed with `vgg16` to avoid confliction,
# load params from `models` rather than `models/vgg16`
fluid.io.load_params(exe, "models") 
```
instead of
```
import paddle.fluid as fluid
exe = fluid.Executor(fluid.CPUPlace())
program, feed_names, fetch_targets = fluid.io.load_inference_model(
                                        './VGG19_pd_model_param', 
					exe, 'vgg19_model', 'vgg19_params')
```
because I dont want another program,vgg19_loss is a part of the g_program,which loss is
```
g_loss = mse_loss + vgg_loss + g_gan_loss
```
I am reimplementing the SRGAN...

**Comment by toddwyl** [comment]
所以我现在需要自定义一个VGG19的接口，他的接口太固定太死板了，我写好了VGG19的接口但是参数名对不上他原来用命令`python vgg19.py vgg19.npy ./fluid`这里的vgg19也是用caffe2fluid生成的，这样生成的params对不上我简化了的接口,想问问你怎么做的？

**Comment by toddwyl** [comment]
I fix this problem in [vgg19_paddle](https://github.com/strawsuper/vgg19_paddlepaddle).:+1:
